ops/testing.py

```diff
@@ -363,7 +365,47 @@ def set_leader(self, is_leader: bool = True) -> None:
         if is_leader and not was_leader and self._charm is not None and self._hooks_enabled:
             self._charm.on.leader_elected.emit()
 
+    def _get_backend_calls(self, reset=True):
+        """Return the calls that we have made to the TestingModelBackend.
+
+        This is useful mostly for testing the framework itself, so that we can assert that we
+        do/don't trigger extra calls.
+
+        :param reset: If True, reset the calls list back to empty, if false, the call list is
+            preserved.
+        :type reset: bool
+        :return: [(call1, args...), (call2, args...)]
```

Comment:
Should this be updated to the new format? It's an honest question, as this is NOT a public method... Wonder if we need to even define params and everything...

Reply:
Any docs that we do produce should certainly be in the new format.
Do we *have* to document our internal functions, no. Should we when possible, yes. Having documentation helps *us* understand what is going on.

But it doesn't end up in the rendered docs.